tests/mcp/src/test/java/io/helidon/extensions/mcp/tests/AbstractMcpSdkLoggingTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.extensions.mcp.tests;
+
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+import io.helidon.webserver.http.HttpRouting;
+import io.helidon.webserver.testing.junit5.SetUpRoute;
+
+import io.modelcontextprotocol.client.McpSyncClient;
+import io.modelcontextprotocol.spec.McpSchema;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.isOneOf;
+
+abstract class AbstractMcpSdkLoggingTest {
+
+    abstract McpSyncClient client();
+
+    abstract CountDownLatch latch();
+
+    @SetUpRoute
+    static void routing(HttpRouting.Builder builder) {
+        LoggingNotifications.setUpRoute(builder);
+    }
+
+    @AfterEach
+    void afterEach() throws InterruptedException {
+        assertThat(latch().await(20, TimeUnit.SECONDS), is(true));
+    }
+
+    @Test
+    void testMcpSdkProgress() {
+        client().callTool(new McpSchema.CallToolRequest("logging", Map.of("question", "")));
```

Comment:
Because of https://modelcontextprotocol.io/specification/2025-03-26/server/tools#error-handling, this test won't pass in the future.
Tool input schema is empty. So calling this tool with an argument `question` will result in an `Invalid Argument` Error. Today the test passes because we do not validate the inputs against the schema yet. But I would be proactive and use `Map.of()` here.

Reply:
Yes, good point. I can create an issue, we also need to convert all other McpSdk tests to run with streamable HTTP.